TSOClient/tso.simantics/primitives/VMTestObjectType.cs

```diff
@@ -22,7 +22,8 @@ public override VMPrimitiveExitCode Execute(VMStackFrame context)
                 return VMPrimitiveExitCode.ERROR;
             }
 
-            if (operand.GUID == 0xDC6D7898) operand = operand;
+            //What's the point of this statement?
```

Comment:
So that I can breakpoint when it hits a certain type of object. Does nothing functionally, only really used by me debugging.